src/main/java/util/Storage.java

```diff
@@ -0,0 +1,90 @@
+package util;
+import java.io.FileWriter;
```

Comment:
Should your import statements be consistent throughout the project, some have one-line spacings between the package and import, while some do not.

Reply:
Good catch, didn't think of that!